enterprise/views.py

```diff
@@ -175,9 +186,12 @@ def get_default_context(self, enterprise_customer, platform_name):
             'confirmation_modal_abort_decline_text': self.modal_abort_decline_msg,
             'policy_link_template': self.policy_link_template,
             'policy_return_link_text': self.policy_return_link_text,
-            'LMS_SEGMENT_KEY': settings.LMS_SEGMENT_KEY,
         }
 
+        global_context_data = get_global_context(request)
+        context_data.update(global_context_data)
+        return context_data
```

Comment:
You could also do to simplify a bit:
```
context_data = get_global_context(request)
context_data.update({
              'page_title': self.page_title,
              'consent_message_header': self.consent_message_header,
              'requested_permissions_header': self.requested_permissions_header.format(
                 enterprise_customer_name=enterprise_customer.name
             ),
             'agreement_text': self.agreement_text.format(
                 enterprise_customer_name=enterprise_customer.name,
                 platform_name=platform_name,
             ),
             'continue_text': self.continue_text,
             'abort_text': self.abort_text,
             'policy_dropdown_header': self.policy_dropdown_header,
             'sharable_items_header': self.sharable_items_header.format(
                 enterprise_customer_name=enterprise_customer.name
             ),
             'sharable_items': [
                 item.format(
                     enterprise_customer_name=enterprise_customer.name,
                     platform_name=platform_name
                 ) for item in self.sharable_items
             ],
             'sharable_items_footer': self.sharable_items_footer.format(
                 enterprise_customer_name=enterprise_customer.name,
                 platform_name=platform_name,
             ),
             'sharable_items_note_header': self.sharable_items_note_header,
             'sharable_items_notes': [
                 item.format(
                     enterprise_customer_name=enterprise_customer.name,
                     platform_name=platform_name
                 ) for item in self.sharable_items_notes
             ],
             'confirmation_modal_header': self.confirmation_modal_header,
             'confirmation_modal_affirm_decline_text': self.modal_affirm_decline_msg,
              'confirmation_modal_abort_decline_text': self.modal_abort_decline_msg,
              'policy_link_template': self.policy_link_template,
              'policy_return_link_text': self.policy_return_link_text,
})
return context_data

Reply:
Actually, ignore this if you decide to do the cleanup of the arguments I mentioned above.